index.html

```diff
@@ -885,6 +831,103 @@ <h5>Timing Properties</h5>
               The end time of a <a>DAPT Script</a> is for practical purposes the end of the <a>Related Media Object</a>.
             </p>
           </section>
+
+          <section id="sharedprop-represents">
+            <h5>Represents</h5>
+            <p>The <dfn>Represents</dfn> property
+              indicates which components of the <a>related media object</a>
+              the contents of the parent object represent.
+              The contents of the object could be used as part of a mechanism
+              to provide an accessible alternative for those components.
+            </p>
+            <p>When applied to a <a>DAPT Script</a>,
+              the contents of the document could be used as part of a mechanism
+              to provide an accessible alternative for those components.</p>
+            <p>When applied to a <a>Script Event</a>,
+              it describes the components of the related media that the
+              <a>Script Event</a>'s <a>Text</a> object(s) represent,
+              during that event's interval.</p>
+            <p>To represent this property,
+              the <dfn><code>daptm:represents</code></dfn> attribute is present on the element
+              corresponding to the parent object,
+              with a value conforming to the following syntax:</p>
+            <div class="exampleInner">
+              <pre class="language-abnf">
+    daptm:represents
+    : &lt;content-descriptor&gt; ( &lt;lwsp&gt;+ &lt;content-descriptor&gt;)*
+  
+    &lt;content-descriptor&gt;  # see registry table below
+  
+    &lt;lwsp&gt;                # as TTML2
+              </pre>
+            </div>
+            <p>The permitted values for <code>&lt;content-descriptor&gt;</code>
+              are listed in the following <a>registry table</a>:</p>
+              <div class="registry-table-section">
+                <table class="data">
+                  <caption><a>Registry table</a> for the <code>&lt;content-descriptor&gt;</code> component
+                    whose Registry Definition is at <a href="#registry-table-content-descriptor"></a></caption>
+                  <thead>
+                    <th><code>&lt;content-descriptor&gt;</code></th>
+                    <th>Status</th>
+                    <th>Description</th>
+                    <th>Used in</th>
```

Comment:
What does "used in" mean? What are the normative implications, if any? Should we say "Example use case" to make it clear that has no normative meaning?

Reply:
Changed to Example Usage.